### DISCUSSION TITLE
Version check error

### DISCUSSION THREAD

**Discussion by Abhis2004** [saran-surya/email_auth_node#7] [Q&A]
email-auth-node : The term 'email-auth-node' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the 
spelling of the name, or if a path was included, verify that the path is correct and try again.    
At line:1 char:1
+ email-auth-node --version
+ ~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (email-auth-node:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Comment by saran-surya** [comment]
Hi @Abhis2004 
Kindly try using ```npx email-auth-node``` in your command prompt.

**Reply by saran-surya** [reply]
Issue resolved / Issue from user.
